mailpoet/assets/js/src/newsletters/types.tsx

```diff
@@ -103,12 +103,48 @@ function NewsletterTypesComponent({
     );
   };
 
+  const getRedirectToAutomateWooType = () => {
+    const redirectToAutomateWoo = (): void => {
+      MailPoet.trackEvent('Emails > Type selected', {
+        'Email type': 'woocommerce_automatewoo',
+      });
+      window.location.href = `edit.php?post_type=aw_workflow#presets`;
+      return null;
+    };
```

Comment:
Event tracking is async, so with this implementation, the redirect may happen before the event is sent to Mixpanel. So I think not all events would be sent.

Here is the code that should ensure sending the event before the redirect
```JS
MailPoet.trackEvent(
    'Emails > Type selected',
    {
        'Email type': 'woocommerce_automatewoo',
     },
    { send_immediately: true }, // This tell Mixpanel client to send events from buffer immediately
    () => { // Anonymous callback which does the redirect
      window.location.href = `edit.php?post_type=aw_workflow#presets`;
    },
  );
```